pupil-spa/src/app/instructions/instructions.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import {Router} from '@angular/router';
+
+import { InstructionsComponent } from './instructions.component';
+
+describe('InstructionsComponent', () => {
+  let component: InstructionsComponent;
+  let fixture: ComponentFixture<InstructionsComponent>;
+  let mockRouter;
+
+  beforeEach(async(() => {
+    mockRouter = {
+      navigate: jasmine.createSpy('navigate')
```

Comment:
There no is point in creating a spy unless you actually want to spy on something.  Could replace with empty function, or check it has actually been called.